src/TimeTravel/TimeTravel.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import { IconSvg } from "../IconSvg";
+import "./TimeTravel.scss";
+import { ITimeTravelProps as Props } from "./types";
+
+export const TimeTravel: React.FC<Props> = ({ onDatesChange }) => {
+	const handleAddMonth = () => onDatesChange("+m");
+	const handleSubtractMonth = () => onDatesChange("-m");
+	const handleAddWeek = () => onDatesChange("+w");
+	const handleSubtractWeek = () => onDatesChange("-w");
+	const handleAddDay = () => onDatesChange("+d");
+	const handleSubtractDay = () => onDatesChange("-d");
```

Comment:
Вполне можно было обойтись одной функцией
```suggestion
	const handleDatesChange = (val: AllowedDateChanges) => onDatesChange(val);
```